vignettes/shared_cache.Rmd

```diff
@@ -8,45 +12,154 @@ vignette: >
   %\VignetteEngine{knitr::rmarkdown}
   \usepackage[utf8]{inputenc}
 ---
+<head>
+<style>
+data {color: #CA3542;}
+ind {color: #FEBC38;}
+build {color: #697F98;} 
+target {color: #37AFA9}
+</style>
+</head>
+
   
 ```{r setup, include=FALSE}
 knitr::opts_chunk$set(echo = TRUE, cache=FALSE, collapse=TRUE)
-knitr::opts_knit$set(root.dir = tempdir())
 ```
 
-## Setup
 
-```{r libs, message=FALSE}
-library(scipiper)
-library(dplyr)
-```
+### Terms and concept
 
-## Shared cache
+####Terms
 
-A shared cache is a cloud data storage location where raw, intermediate, and/or final data products from an analysis project are contributed to and accessible by multiple analysts. Not all scipiper projects will use a shared cache.
+* <data>**Data file**</data>: files used as input for or generated as output by computing (e.g. streamgage data, geospatial data, site metadata, modeling results) 
+* <ind>**Indicator file**</ind>: git-versioned files that promise the analyst that the data file exists remotely 
+* <build>**Build status file**</build>: git-versioned files that represent the collective project build status (e.g. which targets have been built and/or are up-to-date) 
+* <target>**Target**</target>: points or steps in the analysis that can be files or R objects. Many of the targets in a shared cache project will be <ind>indicator files</ind>. 
 
-### Pros and Cons
+####Concept
 
-Advantages of a shared cache:
-* Not every analyst needs to build every target, saving on total processing time.
-* Targets that can only be built on specific operating systems (e.g., Mac) or in specific computing environments (e.g., a cluster) can still be accessible to all analysts for further analysis.
-* Intermediate and final products can be immediately visible to anyone who has access to the shared cache, whether they are contributing to the analysis or simply inspecting/using the output.
+A **s**hared **c**ache (**sc** of `scmake`) is a cloud data storage location where raw, intermediate, and/or final data products from an analysis project are contributed to and accessible by multiple analysts. Not all scipiper projects will use a shared cache.
+
+<data>**Data files**</data> only _need_ to be local when the analyst is computing with the <data>**data file**</data>. <ind>**Indicator files**</ind> (`.ind`) represent the remote shared cache among project participants. This allows analyst #1 to compute part of the project, upload output in the project's shared cache, and analyst #2 can use the output from analyst #1 without redoing the computing performed by analyst #1 
+
+Workflow dependencies are connected via the <ind>**indicator files**</ind>. Recipes (e.g. R scripts) used to create the <ind>**indicator files**</ind> pull down the <data>**data files**</data> from the shared cache using `scipiper` functions such as `gd_get()`. 
```

Comment:
I'd make a distinction between the creation of an indicator file, which usually involves pushing a data file _to_ the shared cache, from the use of an indicator file, which usually involves pulling a data file _from_ the shared cache (if it's not already available locally).

Reply:
Ah good distinction, I'll add that